homeassistant/components/lametric/config_flow.py

```diff
@@ -249,7 +249,10 @@ async def _async_step_create_entry(
         device = await lametric.device()
 
         if self.source != SOURCE_REAUTH:
-            await self.async_set_unique_id(device.serial_number)
+            await self.async_set_unique_id(
+                device.serial_number,
+                raise_on_progress=False,
```

Comment:
We're missing a test for the case where there's a another flow in progress.

Reply:
Will add one 👍